docs/advanced/classes.rst

```diff
@@ -1232,3 +1232,17 @@ appropriate derived-class pointer (e.g. using
     more complete example, including a demonstration of how to provide
     automatic downcasting for an entire class hierarchy without
     writing one get() function for each class.
+
+Accessing the type object
+=========================
+
+You can get the type object from a C++ class that has already been registered using:
+
+.. code-block:: python
+
+    auto T_py = py::type::of<T>();
```

Comment:
show type here (e.g. `py::type`) just for clarity?

Reply:
It looked like duplicated info, but it's really not, I guess, since py::type::function doesn't necessarily have to return a py::type.

Reply:
It's meant to look like duplicate info, the way we're designing this API, though? ;-)